source/_patterns/50-pages/40-demopages/buttons.twig

```diff
@@ -26,6 +27,11 @@
     </div>
 
     <h3>Pill Buttons</h3>
+    {% include 'atoms-pill-button' with {
+        'styleModifier' : 'pill-button--primary pill-button--themed pill-button--theme-call_to_action',
```

Comment:
call-to-action vs call_to_action
Hai, mit den Namen da haben wir es echt druff

Reply:
Oha, gar nicht gesehen! 😄

Reply:
ouh, und `button--call-to-action` != `pill-button--theme-call_to_action` oder `pill-button--theme-call-to-action`

Reply:
Never mind, ich hab nicht gesehen dass es `pill-button--...` und `button--...` gibt

Reply:
Kebab-Case wär schon passender :)

Reply:
also button--call-to-action und pill-button--theme-call-to-action?

Reply:
dann werden aber die pill-button--themes in sich inkonsistent

Reply:
Stimmt, da müssen wir dann noch aufräumen